js/site-nav.js

```diff
@@ -7,18 +7,20 @@ var $ = require('jquery');
 function SiteNav(selector) {
   this.$body = $(selector);
   this.$toggle = this.$body.find('.js-nav-toggle');
+  this.$openSublist = null;
 
   this.assignAria();
 
   this.$toggle.on('click', this.toggle.bind(this));
-  this.$body.on('change', '.js-toggle', this.handleChange.bind(this));
-  this.$body.on('mouseenter', 'li:has(.js-toggle)', this.handleMouseEnter.bind(this));
-  this.$body.on('mouseleave', 'li:has(.js-toggle)', this.handleMouseExit.bind(this));
+  this.$body.on('click', '.js-sublist-toggle', this.toggleSublist.bind(this));
+  this.$body.on('mouseenter', '.js-sublist-parent', this.toggleSublist.bind(this));
```

Comment:
For clarity, how about binding "mouseenter" to `showSublist` and "mouseleave" to `hideSublist`?